subsys/logging/log_core.c

```diff
@@ -299,6 +299,10 @@ void log_panic(void)
 {
 	struct log_backend const *backend;
 
+	if (panic_mode) {
```

Comment:
Why this is needed?

Reply:
unfortuantely there are many paths in error handling. So i see scenarios where one fault function which called log_panic will call another one and i don't want to multiple panic calls to backends. Note that we want to enter panic mode as soon as possible to get as much logs as possible (flush as early as possible).